noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/partial_note.nr

```diff
@@ -38,12 +41,10 @@ pub fn compute_partial_public_log_payload<let N: u32, let M: u32>(
     // TODO: also use this shared secret for deriving note randomness.
 
     // *****************************************************************************
-    // Prepend/append extra bytes
+    // Compute the plaintext
     // *****************************************************************************
 
-    // "Proper" meaning the main meaty stuff that we care about.
-    let proper_plaintext: [u8; N] = plaintext;
-    let final_plaintext = proper_plaintext;
+    let final_plaintext = plaintext;
```

Comment:
Why this re-assignment? To differentiate it from header plaintext? Seems superfluous

Reply:
Removed :)